src/javascripts/i18n.js

```diff
@@ -0,0 +1,38 @@
+import manifest from '../../dist/manifest.json';
+const defaultLocale = manifest.defaultLocale || 'en';
+
+function tryRequire(locale) {
+  try {
+    return require(`../translations/${locale}.json`);
+  } catch(e) {
+    return null;
+  }
+}
+
+module.exports = Object.freeze({
+  t: function(app, key, context) {
+    var keyType = typeof key;
+    if (keyType !== 'string') {
+      throw new Error(`Translation key must be a string, got: ${keyType}`);
+    }
+    var template = _.at(app.translations, key)[0];
```

Comment:
lodash is included at runtime by jsdelivr rather than by webpack, so no require...